scripts/common/preflights.sh

```diff
@@ -657,3 +658,21 @@ function bail_if_unsupported_openebs_to_rook_version() {
     fi
 }
 
+# not allow run the installer/upgrade when kurl version is lower than the previous applied before
+function bail_if_kurl_version_is_lower_previous_config() {
+    if commandExists kubectl; then
+       local current_kurl_version
+       current_kurl_version=$(kubectl -n kurl get configmap kurl-current-config -o jsonpath='{ .data.kurl-version }')
+       if [ -z "$current_kurl_version" ]; then
+           current_kurl_version=$(kubectl -n kurl get configmap kurl-last-config -o jsonpath='{ .data.kurl-version }')
+       fi
+
+       semverCompare $(echo "$KURL_VERSION" | sed 's/v//g') $(echo "$current_kurl_version" | sed 's/v//g')
+       if [ "$SEMVER_COMPARE_RESULT"  = "-1" ]; then # greater than or equal to 14.2.21
+           logFail "This host was updated or installed using Kubernetes Installer(kURL) release version"
+           logFail "$current_kurl_version which is earlier than the version $KURL_VERSION."
```

Comment:
in my opinion the variable `current_kurl_version` is confusing as named and should be named `previous_kurl_version`

```suggestion
           logFail "$current_kurl_version which is greater than the version $KURL_VERSION."
```